Dockerfile

```diff
@@ -24,6 +24,8 @@ RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
 # Copy the operator binary into a lighter image
 FROM centos:8
 
+RUN yum update --setopt=tsflags=nodocs -y && yum clean all
```

Comment:
Can this lead a build job from the same commit to produce 2 different Docker images, depending on underlying yum packages changes?

Reply:
Absolutely. This is already the case though because the centos:8 tag changes out from under us as it gets updated.